src/lib/svg_text_utils.js

```diff
@@ -278,6 +280,13 @@ function encodeForHTML(_str) {
 function convertToSVG(_str) {
     _str = convertEntities(_str);
 
+    // normalize behavior between IE and others wrt newlines and whitespace:pre
+    // this combination makes IE barf https://github.com/plotly/plotly.js/issues/746
+    // Chrome and FF display \n, \r, or \r\n as a space in this mode.
+    // I feel like at some point we turned these into <br> but currently we don't so
+    // I'm just going to cement what we do now in Chrome and FF
+    _str = _str.replace(NEWLINES, ' ');
```

Comment:
As far as I can tell this is purely a bug in IE - though one that persists from 9 through 11... the affected text is invisible, even though `getBoundingClientRect()` on its node shows the correct position and shape, and it reappears when you take out the newline character.